README.md

```diff
@@ -37,15 +37,15 @@ Contributions are made to this repo via Issues and Pull Requests \(PRs\). A few
 * Search for existing Issues and PRs before creating your own.
 * You're going to edit the file in GitHub after clicking on "edit on github" in the Mumble Docs Website
 
-{% embed url="https://github.com/MidouWebDev/Mumble-docs" caption="Check the Mumble Documentation Repo" %}
```

Comment:
Those {% embed url="..." caption="..." %} where changed to [caption](url) format